source/extensions/filters/http/lua/lua_filter.cc

```diff
@@ -23,7 +90,10 @@ StreamHandleWrapper::StreamHandleWrapper(Filters::Common::Lua::Coroutine& corout
         if (state_ == State::Running) {
           throw Filters::Common::Lua::LuaException("script performed an unexpected yield");
         }
-      }) {}
+      }),
+      noopCallbacks_(new NoopCallbacks()) {}
```

Comment:
Since `NoopCallbacks` doesn't hold any state, I think it would be clearer to define it as a `ConstSingleton` and just pass the reference to the singleton object. That way we avoid having to deal with this memory management.

Reply:
vc'd. created static object instead.